app/controllers/works_controller.rb

```diff
@@ -72,7 +72,10 @@ def show
     respond_to do |format|
       format.html do
         # Ensure that the Work belongs to a Collection
-        raise(Work::InvalidCollectionError, "The Work #{@work.id} does not belong to any Collection") unless @work_decorator.collection
+        @collection = @work_decorator.group
+        raise(Work::InvalidCollectionError, "The Work #{@work.id} does not belong to any Collection") unless @collection
+
+        @can_curate = current_user.can_admin?(@collection)
```

Comment:
This may ripple but I think this should be @group here
```suggestion
        @group = @work_decorator.group
        raise(Work::InvalidCollectionError, "The Work #{@work.id} does not belong to any Group") unless @group

        @can_curate = current_user.can_admin?(@group)
```